Sources/TUSKit/TUSClient.swift

```diff
@@ -103,7 +103,7 @@ public final class TUSClient {
     /// - Important: The client assumes by default that your server implements the Creation TUS protocol extension. If your server does not support that,
     ///   make sure that you provide an empty array in the `supportExtensions` parameter.
     /// - Throws: File related errors when it can't make a directory at the designated path.
-    public init(server: URL, sessionIdentifier: String, storageDirectory: URL? = nil, session: URLSession = URLSession.shared, chunkSize: Int = 500 * 1024, supportedExtensions: [TUSProtocolExtension] = [.creation]) throws {
+    public init(server: URL, sessionIdentifier: String, storageDirectory: URL? = nil, session: URLSession = URLSession.shared, chunkSize: Int = 500 * 1024, supportedExtensions: [TUSProtocolExtension]? = nil) throws {
```

Comment:
I don't think we should be changing this default argument. The change in this PR is mostly additive and can be made in a non-breaking way if we keep the current public API stable

Reply:
I agree with Donny here.

Reply:
Yes, I missed this, I will update PR when I am sure the following question is answered